include/ejabberd_c2s.hrl

```diff
@@ -27,12 +27,13 @@
                 tls_required = false  :: boolean(),
                 tls_enabled = false   :: boolean(),
                 tls_options = [],
+                verify                :: verify_none | verify_peer,
```

Comment:
sasl_verify or something more self-describing good to have.
Should we replace this record with a map eventually?

At least with my client we have `everything else` field with a map for any extra stuff.

Reply:
verify_none & verify_peer are the standard atoms in erlang's ssl and fast_tls. so let it stay standard,
if we want to change record to map - let's do it in another CR. this one is already too big

Reply:
I was talking about `#state.verify` field name.
Values are fine. But field name is very confusing.

Can be

```
sasl_verify    :: verify_none | verify_peer
```

or

```
sasl_ssl_verify :: verify_none | verify_peer
```

or

```
sasl_tls_verify :: verify_none | verify_peer
```